README.md

```diff
@@ -3,3 +3,11 @@
 WIP - iOS/Android shared Rust library
 
 Helpers wanted - if you have Rust experience for iOS/Android app integration, please reach out!
+
+## Currently exposed functions:
+- get_reports
+- post_report
```

Comment:
I'm not sure about adding these, as we'd have to remember to modify the readme every time that we change the interface (and now in particular the api is very unstable). Maybe a link to the headers instead?

Reply:
Alrighty, linked to the headers (and wiki).